### PR TITLE
Glossary updates: alignment to application-context

### PR DESCRIPTION
**Pull request by estelle**
grammar, links, link order, etc. nothing major

### CODE REVIEW

**Comment by bsmth** [comment]
Taking a look 👀

File: files/en-us/glossary/alpha/index.md [hunk 1]
```diff
@@ -4,17 +4,19 @@ slug: Glossary/Alpha
...
-For example, the color `#8921F2` (also described as `rgb(137, 33, 242)` or `hsl(270, 89%, 54)`) is a nice shade of purple. Below you see a small box of that color in the top-left corner and a box of the _same_ color but with an alpha channel set at 0.5 (50% opacity). The two boxes are drawn on top of a paragraph of text.
+For example, the color `#8921F2` (also described as `rgb(137 33 242)` or `hsl(270 89% 54)`) is a nice shade of purple. Below you see a small box of that color in the top-left corner and a box of the _same_ color but with an alpha channel set at 0.5 (50% opacity), `#8921F280`, where `80` is the hexadecimal equivalent of 50%. This color is also described as `rgb(137 33 242 / 0.5)` or `hsl(270 89% 54 / 0.5)`. The two boxes are drawn on top of a paragraph of text.
 
 ![Image showing the effect of an alpha channel on a color.](alpha-channel-example.png)
 
 As you can see, the color without an alpha channel completely blocks the background text, while the box with the alpha channel leaves it visible through the purple background color.
 
 ## See also
 
+- [CSS color](/en-US/docs/Web/CSS/CSS_colors)
```
Comment (bsmth) [inline files/en-us/glossary/alpha/index.md:17]: minor one:

```suggestion
- [CSS colors](/en-US/docs/Web/CSS/CSS_colors)
```

**Review by bsmth — APPROVED** [review]
A few comments to consider, but leaving a +1 in advance. Thanks @estelle 🙌🏻

File: files/en-us/glossary/alignment_container/index.md [hunk 2]
```diff
@@ -8,4 +8,15 @@ The **alignment container** is the rectangle that the [alignment subject](/en-US
...
+- CSS {{CSSXref("align-items")}} property
+- CSS {{CSSXref("align-self")}} property
+- CSS {{CSSXref("align-tracks")}} property
+- CSS {{CSSXref("justify-content")}} property
+- CSS {{CSSXref("justify-items")}} property
+- CSS {{CSSXref("justify-self")}} property
+- CSS {{CSSXref("justify-tracks")}} property
+- CSS {{CSSXref("place-content")}} shorthand property
+- CSS {{CSSXref("place-self")}} shorthand property
+- CSS {{CSSXref("scroll-snap-align")}} property
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/glossary/alignment_container/index.md:22]: it's basically a list of all the pages where this glossary term is relevant.

**Pull request merged**